docs/findings/el7.md

```diff
@@ -7,199 +7,89 @@
+## Enable FIPS Mode in GRUB2
 
-Note: for programs that place the additional requirement that `/var/tmp` _must_ be a standalone filesystem, it is recommended to up the size of the root volume-group, create an LVM volume to host the `/var/tmp` filesystem and then apply the site-local mount-options to the filesystem. This can all be accomplished at system provisioning-time, but is outside the scope of the watchmaker utility to effect.
+Conditionally-Valid Finding: Both spel and watchmaker implement `fips=1` by default. If finding occurs, either:
 
-## Missing Audit Rules
+* There is an error in scanner's validation-method
+* System has been intentionally de-configured for FIPS &mdash; typically due to hosted-software's requirements &mdash; and will need to be POA&Med.
 
-Some third-paty scanners will incorrectly indicate that the `auditd` subsystem is missing rules.
+## Configure AIDE to Use FIPS 140-2 for Validating Hashes
 
-### Ensure Kernel Module (Un)Loading Information Is Collected
+Valid Finding: [Issue](https://github.com/plus3it/ash-linux-formula/issues/284) submitted to relevant sub-project for fix
 
-Already implemented in the `/etc/audit/rules.d/modules.rules` and `/etc/audit/audit.rules` files. Watchmaker-applied rule uses `-k` to signify the audit event-key. Complaining scanner-utility is likely searching for the `key=` token to signify the audit event-key or similarly-incompatible search-string.
+## Verify and Correct Ownership with RPM
 
-### Ensure Attempts to Alter Logon and Logout Events Is Collected
+Invalid Finding:
 
-Already implemented in the `/etc/audit/audit.rules` and `/etc/audit/rules.d/logins.rules` files. Watchmaker-applied rule uses `-k` to signify the audit event-key. Complaining scanner-utility is likely searching for the `key=` token to signify the audit event-key or similarly-incompatible search-string.
+* Flags on system-journal ownership: Journal ownership settings are automatically reset by systemd (upon reboot) after hardening has run. Currently, no means of permanently remediating is possible.
+* Similarly, if HBSS or VSEL is installed, scan may flag on user-ownership depending on how site specifies installation of HBSS or VSEL. One would reasonably expect similar for other, third-party packages.  "Fixing" (per STIG guidance) would likely break the functioning of the HBSS/VSEL (or third-party) software
```

Comment:
I think we should *focus* on the general cause here (installing packages using `rpm` instead of `yum`), and mention the McAfee Agent as a specific example, rather than the other way around.

Reply:
Using `rpm` vice `yum` isn't the issue, per se. It's a combination of:

* The relevant RPMs not declaring certain files as "config" (or even "ghost") files
* The scanner is using a faulty change-detection method - ignoring files that actually _are_ managed as "config" (or "ghost") files

Reply:
HBSS isn't actually a package, right? It's really the McAfee Agent and McAfee VSEL?